examples/lines_bars_and_markers/bar_stacked.py

```diff
@@ -21,14 +21,16 @@
 ind = np.arange(N)    # the x locations for the groups
 width = 0.35       # the width of the bars: can also be len(x) sequence
 
-plt.bar(ind, men_means, width, yerr=men_std, label='Men')
-plt.bar(ind, women_means, width, yerr=women_std, bottom=men_means,
+fig, ax = plt.subplots()
+
+ax.bar(ind, men_means, width, yerr=men_std, label='Men')
+ax.bar(ind, women_means, width, yerr=women_std, bottom=men_means,
         label='Women')
```

Comment:
reindent this line too.
anyone can merge postci

Reply:
done.